src/makeInfinityAdder.js

```diff
@@ -4,7 +4,25 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let result = 0;
+
+  const infinityAdder = (firstNumber) => {
+    const saveResult = [];
+
+    if (firstNumber !== undefined) {
+      result += firstNumber;
+
+      return infinityAdder;
+    } else {
+      saveResult.push(result);
+
+      result = 0;
+
+      return saveResult[0];
```

Comment:
Instead of pushing the result to `saveResult` and then returning the first element, you can return the `result` directly and then reset it.

Reply:
Consider fixing AI comments